apps/sms/views/conversation/js/conversation.js

```diff
@@ -692,11 +692,14 @@ var ConversationView = {
     // Add recipients and fill+focus the Compose area.
     var recipientPromise;
     if (activity.contact && number) {
-      recipientPromise = Utils.getContactDisplayInfo(findByAddress, number)
-        .then(function onData(data) {
-          data.source = 'contacts';
-          return data;
-        });
+      findByAddress(phoneNumber).then((contacts) => {
```

Comment:
note that if you install jshint locally, it should tell it to you once you configure it with your text editor :)